CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Release Notes
+
+## v?
```

Comment:
Please let me know what the appropriate version is.

Reply:
3.0